lib/charms/magma_orchestrator_interface/v0/magma_orchestrator_interface.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Library for the magma-orchestrator relation.
+
+This library contains the Requires and Provides classes for handling the magma-orchestrator
+interface.
+
+## Getting Started
+From a charm directory, fetch the library using `charmcraft`:
+
+```shell
+charmcraft fetch-lib charms.magma_orchestrator_interface.v0.magma_orchestrator_interface
+```
+
+Add the following libraries to the charm's `requirements.txt` file:
+- jsonschema
+
+### Requirer charm
+The requirer charm is the charm requiring to connect to an orchestrator from another charm that
```

Comment:
Should we specify that we're talking about Magma Orchestrator here?

Reply:
Done